### PR TITLE
Have schema checkbox labels on the right

### PR DESCRIPTION
**Pull request by kennyt**
depends on https://github.com/dcos/dcos-ui/pull/693

# Before
px

# After
px

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @philipnrmn

File: src/js/utils/SchemaUtil.js [hunk 1]
```diff
@@ -26,13 +26,12 @@ function setLabelFromSchemaProperty(fieldName, fieldProps, isRequired, renderLab
   }
 
   if (renderLabel && fieldProps.description) {
-    label = renderLabel(fieldProps.description, label);
+    label = renderLabel(fieldProps.description, label, fieldProps.type);
   }
 
   // Set the label property of checkboxes if a label is defined.
-  if (fieldProps.type === 'boolean' && fieldProps.label != null
-    && fieldProps.label !== '') {
```
Comment (rcorral) [inline src/js/utils/SchemaUtil.js]: Why did you remove the last two checks?

Comment (kennyt) [inline src/js/utils/SchemaUtil.js]: because even if there are no `label` property on the schema, we can still have a label from the `title` property, which we set above
